pkg/controller.v1beta2/tensorflow/tensorflow.go

```diff
@@ -88,6 +90,7 @@ func genTFConfigJSONStr(tfjob *tfv1beta2.TFJob, rtype, index string) (string, er
 			Type:  rtype,
 			Index: int(i),
 		},
+		RpcLayer: tfjob.Spec.TFRpcLayer,
```

Comment:
Can we keep it empty if tfjob.Spec.TFRpcLayer is nil?

Reply:
I think we can keep it empty. 

In tensorflow:

```
rpc_layer: (String, optional) Overrides the rpc layer TensorFlow uses.
```

https://github.com/tensorflow/tensorflow/blob/5beabd0c7d0d4dc3610360b1df4dfa55111577f4/tensorflow/python/distribute/cluster_resolver/tfconfig_cluster_resolver.py#L67

The default value of it is 'grpc'.